iree/compiler/Dialect/Flow/Transforms/DispatchConfig.cpp

```diff
@@ -21,13 +21,35 @@
 #include "mlir/Dialect/StandardOps/IR/Ops.h"
 #include "tensorflow/compiler/mlir/xla/ir/hlo_ops.h"
 
-#define DEBUG_TYPE "iree-dispatch-config"
+#define DEBUG_TYPE "xxx"
 
 namespace mlir {
 namespace iree_compiler {
 namespace IREE {
 namespace Flow {
 
+namespace {
+// TODO(laurenzo): Every one of these should have better support and removed
+// from this exclusion list eventually.
+bool isUnsupportedFusionOp(Operation *op) {
+  return isa<xla_hlo::DotOp>(op) || isa<xla_hlo::ConvOp>(op) ||
+         isa<xla_hlo::ReduceOp>(op) || isa<xla_hlo::PadOp>(op) ||
+         isa<xla_hlo::ReduceWindowOp>(op);
+}
+
+// Whitelist of ops that materialize to a (mapped) copy of some kind if they
+// exist standalone. Generally we try to avoid anchoring on these, letting them
+// fuse into more meaningful ops as possible.
+bool isCopyOp(Operation *op) {
```

Comment:
this should be 'indexed'/'indexing' instead of copy - copy ops (like clone or update-slice) are different and things we may want to call out specially (to ensure they don't get folded at all so they remain visible to the scheduler, instead of those like these that should be aggressively fused)

probably want gather in here as well